lib/Conf.ml

```diff
@@ -1243,8 +1251,15 @@ let kind : Syntax.t option ref =
   let intf = (Some Syntax.Signature, Arg.info ["intf"] ~doc ~docs) in
   let doc = "Deprecated. Same as $(b,impl)." in
   let use_file = (Some Syntax.Use_file, Arg.info ["use-file"] ~doc ~docs) in
+  let doc =
+    "Parse file with unrecognized extension as toplevel phrases with their \
```

Comment:
I think the "unrecognized extension" part is misleading because this option takes priority even if the extension would be recognized as something else. (same remark for `impl` and `intf`)

Reply:
True, I will fix this in another PR in the meantime.

Reply:
done